### ISSUE TITLE
Training Configuration for ActivityNet

### ISSUE THREAD

**Issue by PeijunBAO** [crodriguezo/TMLGA#4]
Hi, I am very interested in your code. But I have two question about training configuration for activitynet.
1) Is the training configuration identical to charades ?(e.g. network achetecture, batch size, learning rate)
2) Could you please upload training configuration file i.e. activitynet_train.yaml first? (Instead of large feature file)?
Thank you very much?
